server/server.js

```diff
@@ -26,8 +26,15 @@ app.get('/api/v1/entries', async (req, res) => {
 });
 
 app.post('/api/v1/entries', async (req, res) => {
+  const newEntry = req.body;
+
+  for (let parameter of ['user_id', 'date', 'feeling', 'activity', 'journal_entry']) {
```

Comment:
This is really cool, I'd love to play around with this method sometime, haven't seen it used much.

Reply:
Such a clean way to handle for missing data!